tests/iteminfo_test.cpp

```diff
@@ -620,6 +620,74 @@ TEST_CASE( "weapon attack ratings and moves", "[iteminfo][weapon]" )
                "  Vs. Agile: <color_c_yellow>2.46</color>"
                "  Vs. Armored: <color_c_yellow>0.00</color>\n" );
     }
+
+    SECTION( "stamina per swing" ) {
+        // Stamina cost per swing is dynamically calculated based on many different 
```

Comment:
These tests are a little fragile for my liking, but you probably already knew that. My suggestion would be to either check the numbers directly instead, make test items for your tests, or both.